docs/contributor-docs/release-procedure.md

```diff
@@ -0,0 +1,88 @@
+# Release Procedure
+
+This document is a Core team guide, provided publicly to ensure transparancy in the release process.
+
+## Versioning
+
+Jellyfin uses [semantic versioning](https://semver.org). All releases will have versions in the `X.Y.Z` format, starting from `10.0.0`. Note however that the `10.Y.Z` release chain represents the "cleanup" of the codebase, so it should be accepted that `10.Y.Z` breaks all compatibility, at some point, with previous Emby-compatible interfaces. Our versioning will typically follow the patterns below:
+
+#### `X` - Major breaking versions
+
+* Breaks compatibility with the HTTP and/or plugin APIs.
+
+#### `Y` - Minor versions
+
+* Introduces new features.
+* Makes minor non-breaking API changes.
+
+#### `Z` - Hotfix versions
+
+* Introduces critical bugfixes or otherwise changes `master` branch code since the last release.
+
+## General Release Philosophy
+
+## Release procedure (major/minor version releases)
+
+1. Announce release in the [jellyfin-dev](https://matrix.to/#/#jellyfin-dev:matrix.org) Matrix/Riot channel at least a few hours before releasing, to put a temporary freeze on merges.
+
+1. Ensure any required PRs are merged into both the [jellyfin](https://github.com/jellyfin/jellyfin) and [jellyfin-web](https://github.com/jellyfin/jellyfin-web) repositories.
+
+1. Create a release branch for the [jellyfin](https://github.com/jellyfin/jellyfin) and [jellyfin-web](https://github.com/jellyfin/jellyfin-web) repositories with the format `release-X.Y.Z`, where `X.Y` is the new minor + major version number and `Z` is a literal `Z` character, based off the current `master` branches. These will be somewhat long-lived branches to track particular releases and deal with hotfixes to those branches should they be required.
+
+1. Execute the `bump_version` script inside the release branch in the [jellyfin](https://github.com/jellyfin/jellyfin) local repository. Commit the resulting differences as `Bump version to X.Y.Z`, where `X.Y.Z` is the full version number.
+
+1. Perform initial testing builds and test the resulting binaries.
+
+    Basic testing checklist:
+
+    1. Run through setup wizard and import small library.  
+    1. Test playback, navigation, and subtitles in the Web UI.  
+    1. Test playback, navigation, and subtitles in the Chromecast app.  
```

Comment:
```suggestion
    1. Test playback, navigation, and subtitles in the Chromecast app and other released clients.  
```

Reply:
Replaced the explicit Chromecast mention with "various client apps". Fixing.